client/rpc-api/src/system/error.rs

```diff
@@ -43,6 +45,11 @@ impl From<Error> for rpc::Error {
 				message: format!("{}", e),
 				data: serde_json::to_value(h).ok(),
 			},
+			Error::MalformattedPeerArg(ref e) => rpc::Error {
+				code :rpc::ErrorCode::ServerError(BASE_ERROR + 2),
+				message: format!("{}", e),
```

Comment:
already a `String` no need for `format!`

```suggestion
				message: e.clone(),
```